rotkehlchen/exchanges/kraken.py

```diff
@@ -178,6 +191,9 @@ def __init__(
             database: 'DBHandler',
             msg_aggregator: 'MessagesAggregator',
             kraken_account_type: KrakenAccountType | None = None,
+            kraken_futures_api_key: ApiKey | None = None,
+            kraken_futures_api_secret: ApiSecret | None = None,
+            futures_base_uri: str = KRAKEN_FUTURES_BASE_URL,
```

Comment:
since this is already a constant just use it directly. no need to pass it as a parameter here.